.github/continuous-integration.sh

```diff
@@ -40,7 +40,7 @@ if [[ -d "${SCCACHE_DIR}" ]]; then
     echo "It contains $(du -hs ${SCCACHE_DIR} | cut -f1)"
 else
     echo "Creating Sccache directory at ${SCCACHE_DIR}"
-    mkdir -p ${SCCACHE_DIR}
+    sudo mkdir -p ${SCCACHE_DIR}
```

Comment:
This hung before and is fishy. If the build doesn't run as root and you create the sccache dir as root, the build will not be allowed to write to it (unless you change permissions or ownership). Also there should be no need for sudo as `${SCCACHE_DIR}` is in the arc-runners home directory (are files/directories in the home directory owned by root due to a previous run where the user was still root?).

Reply:
Hmm, before it did not compile without sudo. I can try remove it and see what happens

Reply:
wait, huh, I can't find these changes

Reply:
Me neither, looks like I reviewed something stale.

Reply:
It's gone now, so LGTM.